### ISSUE TITLE
Menu bars: Different behavior b/t hover-menu link and full-menu link button

### ISSUE THREAD

**Issue by metasj** [pubpub/pubpub#605]
As an editor, I have a URL in my paste buffer, select a bit of text, click the link icon to add a link, and paste the URL once a text field appears.

If clicking on the link icon in the hovering grey bar that appears after a selection, it pops up a textfield below it, as well as to the right of the Media menu, and the paste works as expected. 

If clicking the link icon in the toolbar above the editing area, it only opens a textfield to the right of the Media menu, and a paste replaces the selected text instead. 

When working quickly it's easy to mistake which menu was used, and the different behavior is confusing.

**Comment by gabestein** [comment]
Yup, still an issue for me, too. The inline mini editor link button's behavior is unpredictable. The first click on it after highlighted text does not pop up the hovering grey bar for adding links. Subsequent clicks seem to. Would be nice to iron this one out.

**Comment by metasj** [comment]
When I first filed this, it reliably worked on the first click if I chose the link in the inline mini editor. Later on (by the time #693 was filed) this changed.

Now this seems better, I haven't run across the problem in a while.
* Ctrl-K almost? always works as expected, but isn't an option on mobile. 
* All workflows that should open a url text-field do add an underline and make the text clickable.  Currently as a workaround, if the URL field doesn't appear after a first try, I click on the text to make it appear.
